src/workerd/server/workerd.c++

```diff
@@ -1102,7 +1191,14 @@ private:
     pid_t pid = getpid();
   	char pathbuf[PROC_PIDPATHINFO_MAXSIZE];
     if (proc_pidpath(pid, pathbuf, sizeof(pathbuf)) > 0) {
-      return tryOpenExe(pathbuf);
+      return tryOpenExe(fs, pathbuf);
+    }
+  #endif
+
+  #if _WIN32
+    char pathbuf[MAX_PATH];
+    if (GetModuleFileNameA(NULL, pathbuf, MAX_PATH) > 0) {
```

Comment:
Will this break if the path contains Unicode characters? Do we need to use the W version and convert to UTF-8? (KJ has functions for this in `kj/encoding.h`.)